### ISSUE TITLE
Utils.validateFile() breaks if checksum algorithm is not defined

### ISSUE THREAD

**Issue by luis100** [keeps/commons-ip#13]
If checksum algorithm or checksum is not defined give a proper error in validation issues.

**Comment by luis100** [comment]
```
java.lang.NullPointerException: null
	at java.security.Provider$ServiceKey.<init>(Provider.java:872)
	at java.security.Provider$ServiceKey.<init>(Provider.java:865)
	at java.security.Provider.getService(Provider.java:1039)
	at sun.security.jca.ProviderList.getService(ProviderList.java:332)
	at sun.security.jca.GetInstance.getInstance(GetInstance.java:157)
	at java.security.Security.getImpl(Security.java:695)
	at java.security.MessageDigest.getInstance(MessageDigest.java:167)
	at org.roda_project.commons_ip.utils.Utils.calculateChecksum(Utils.java:149)
	at org.roda_project.commons_ip.utils.Utils.validateFile(Utils.java:178)
	at org.roda.core.plugins.plugins.ingest.eupo.ip.EUPOSIP.processDivDescriptiveMetadata(EUPOSIP.java:512)
	at org.roda.core.plugins.plugins.ingest.eupo.ip.EUPOSIP.processExpressionManifestations(EUPOSIP.java:567)
	at org.roda.core.plugins.plugins.ingest.eupo.ip.EUPOSIP.processWorkExpressions(EUPOSIP.java:548)
	at org.roda.core.plugins.plugins.ingest.eupo.ip.EUPOSIP.processWorkAndExpressions(EUPOSIP.java:417)
	at org.roda.core.plugins.plugins.ingest.eupo.ip.EUPOSIP.parseEUPOSIP(EUPOSIP.java:351)
	at org.roda.core.plugins.plugins.ingest.eupo.ip.EUPOSIP.parse(EUPOSIP.java:200)
	at org.roda.core.plugins.plugins.ingest.eupo.ip.EUPOSIP.parse(EUPOSIP.java:205)
	at org.roda.core.plugins.plugins.ingest.eupo.EUPOSIPToAIPPlugin.transformTransferredResourceIntoSeveralAIPs(EUPOSIPToAIPPlugin.java:109)
	at org.roda.core.plugins.plugins.ingest.eupo.EUPOSIPToAIPPlugin.execute(EUPOSIPToAIPPlugin.java:98)
	at org.roda.core.plugins.plugins.ingest.DefaultIngestPlugin.transformTransferredResourceIntoAnAIP(DefaultIngestPlugin.java:299)
	at org.roda.core.plugins.plugins.ingest.DefaultIngestPlugin.execute(DefaultIngestPlugin.java:147)
	at org.roda.core.plugins.orchestrate.akka.AkkaWorkerActor.handlePluginExecuteIsReady(AkkaWorkerActor.java:49)
	at org.roda.core.plugins.orchestrate.akka.AkkaWorkerActor.onReceive(AkkaWorkerActor.java:35)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:165)
	at akka.actor.Actor$class.aroundReceive(Actor.scala:484)
	at akka.actor.UntypedActor.aroundReceive(UntypedActor.scala:95)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:526)
	at akka.actor.ActorCell.invoke(ActorCell.scala:495)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:257)
	at akka.dispatch.Mailbox.run(Mailbox.scala:224)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:234)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

**Comment by hsilva-keep** [comment]
Fixed in 0eca70c2aa89d879b23abf91cc2ecbddb4a3d01e

**Comment by jmaferreira** [comment]
Why not set a default one (hardcoded)?
